feature/settings/src/main/java/com/google/jetpackcamera/settings/ui/SettingsComponents.kt

```diff
@@ -235,6 +236,12 @@ fun FlashModeSetting(
                     enabled = true,
                     onClick = { setFlashMode(FlashMode.OFF) }
                 )
+                SingleChoiceSelector(
```

Comment:
low light boost is not supported on all devices? 

How can we verify that a device supports low light boost? Would need to check for that and add a condition to disable this option.

This PR is mostly refactor and UI, and the constraint doesn't affect other settings. so now we could just file a bug and write a TODO.

Reply:
Added a TODO. This will be integrated with #236